eng/templates/stages/release.yml

```diff
@@ -45,6 +45,8 @@ stages:
     value: dotnet
   - name: announcementRepo
     value: source-build
+  - name: publicVmrUrl
+    value: https://github.com/dotnet/dotnet
```

Comment:
`dotnet/dotnet` is already used here: https://github.com/dotnet/source-build/pull/3638/files#diff-16da42ff0ed618ecf203753bec7e4ebd9c976c28ff0753b0807d1d890d3a05cdR202

I would suggest defining a new variable for `dotnet/dotnet` and reference it from there. Then define another derived variable that adds the `https://github.com/` part.

Reply:
Thanks for the suggestion, will improve